src/ol_infrastructure/applications/ocw_site/__main__.py

```diff
@@ -0,0 +1,80 @@
+from pulumi import Config, export
+from pulumi_aws import iam, s3
+from pulumi_vault import aws
+
+from ol_infrastructure.lib.aws.iam_helper import lint_iam_policy
+from ol_infrastructure.lib.ol_types import AWSBase
+from ol_infrastructure.lib.pulumi_helper import parse_stack
+
+ocw_site_config = Config("ocw_site")
+stack_info = parse_stack()
+aws_config = AWSBase(
+    tags={
+        "OU": "open-courseware",
+        "environment": f"applications_{stack_info.env_suffix}",
```

Comment:
The environment tag name should be capitalized.

Reply:
Fixed in f5fc689